src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,34 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.jetbrains.annotations.NotNull;
+
+import javax.persistence.*;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "CARTS")
+public class Cart {
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name = "CART_ID", unique = true)
+    private Long cartId;
+
+    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    @JoinColumn(name = "USER_ID")
+    private User userId;
+
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
CascadeType.ALL for the products attribute's @ManyToMany annotation might cause unwanted side effects such as deleting products when a cart is deleted. Consider using CascadeType.PERSIST instead

Reply:
Good point, I'll change it